scripts/o3de/tests/ExportScripts/test_export_source_built_project.py

```diff
@@ -0,0 +1,1296 @@
+                    mock_bundle_assets.assert_not_called()
+                else:
+
+
+                    selected_tools_build_path = test_tools_build_path if not use_sdk else test_tools_sdk_path
+
+                    if not selected_tools_build_path:
+                        selected_tools_build_path = test_o3de_base_path / 'build/tools'
+                    
+                    if not selected_tools_build_path.is_absolute():
+                        selected_tools_build_path = test_o3de_base_path / selected_tools_build_path
+                    
+                    selected_asset_bundling_path = test_asset_bundling_path if test_asset_bundling_path else \
+                                                        test_o3de_base_path / 'build/asset_bundling'
+                
+                    if not selected_asset_bundling_path.is_absolute():
+                        selected_asset_bundling_path = test_o3de_base_path / selected_asset_bundling_path
+
+                    mock_get_asset_bundler_path.assert_called_once_with(tools_build_path=selected_tools_build_path,
+                                                                    using_installer_sdk=use_sdk,
+                                                                    tool_config=toolconf,
+                                                                    required=True)
+                    
+                    mock_bundle_assets.assert_called_once_with(ctx=mock_ctx,
+                                                            selected_platform=mock_platform,
+                                                            seedlist_paths=[],
+                                                            seedfile_paths=[],
+                                                            tools_build_path=selected_tools_build_path,
+                                                            engine_centric=is_engine_centric,
+                                                            asset_bundling_path=selected_asset_bundling_path,
+                                                            using_installer_sdk=use_sdk,
+                                                            tool_config=toolconf,
+                                                            max_bundle_size=2048)
+
+                mock_get_asset_bundler_path.reset_mock()
+                mock_bundle_assets.reset_mock()
+
+@pytest.mark.parametrize("test_seedlists, test_seedfiles, test_levelnames",[
+    pytest.param([],[],[]),
+    pytest.param([pathlib.Path("C:\\test\\test.seedlist")],[pathlib.Path("C:\\test1\\test.seed")],[]),
+    pytest.param([pathlib.Path("C:\\test\\test.seedlist")],[pathlib.Path("C:\\test1\\test.seed")],["main"]),
+    pytest.param([pathlib.Path("C:\\test\\test.seedlist")],[],["main"]),
+    pytest.param([pathlib.Path("C:\\test\\test.seedlist")],[],["MAIN"]),
+    pytest.param([pathlib.Path("C:\\test\\test.seedlist")],[],["main",'second'])
```

Comment:
Has this been checked to work on Linux.

The pathlib.Path defaults to pathlib.WindowsPath on Widnows and `pathlib.PosixPath` on Linux.

Reply:
This has been verified to work on a clean ubuntu 22.04 machine. Turns out there is no issue with the paths in question. It also works with PurePaths, so I'll follow your suggestion and change it accordingly